dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -454,7 +454,7 @@ class StartupTest {
       final String deviceId = (await devices.workingDevice).deviceId;
       await flutter('packages', options: <String>['get']);
 
-      const int iterations = 3;
+      const int iterations = 15;
```

Comment:
This test takes a while to execute, are you sure you want to run it 15 times?  I think it probably takes at least 1-2 minutes.

Reply:
Yes, I checked the build dashboard and here's what I observed and expected (as documented in the PR/commit description)
> Currently, a startup test runs for about 1 minute. After the change, the
test should run for about 5 minutes or less because some setup steps
don't need to be repeated 5 times.